pages/common/gatsby.md

```diff
@@ -0,0 +1,23 @@
+# gatsby
+
+> Static site generator for React.
+
+- Create a new site:
+
+`gatsby new {{site_name}}`
+
+- Create a new site with a Gatsby 'starter':
+
+`gatsby new {{site_name}} {{url_of_starter_github_repo}}`
+
+- Start a live-reloading local development server:
+
+`gatsby develop`
+
+- Perform a production build and generate static HTML:
+
+`gatsby build`
+
+- Start a local build server:
```

Comment:
What is the difference between this and `gatsby develop` ? Does this serve the prod build and the previous serves the dev build ? If yes, it will be essentially the same as starting any http server and serve the build folder right ?

Reply:
Yes I believe that is correct - does that mean the command is not necessary or should I rephrase it???

Reply:
Hmm yea, IMO its a design flaw from gatsby's side. It should be a single command with `-dev` and `-prod` switches. Perhaps change it to `Start a local server which serves the production build`